client/me/help/help-contact-form/index.jsx

```diff
@@ -99,6 +114,13 @@ export const HelpContactForm = React.createClass( {
 		}
 	},
 
+	doQandASearch() {
+		const query = this.state.subject + ' ' + this.state.message;
+		wpcom.getQandA( query, ( error, data ) => {
```

Comment:
So I added a condition that blanks the search results if there has been an error. We just want to carry on as normal, but without the search results if something went wrong.